website/content/guides/packer-on-cicd/pipelineing-builds.mdx

```diff
@@ -345,8 +345,9 @@ pipeline.sh
 
 ```sh
 #!/bin/bash
+set -o errexit # abort if there is an issue with any build
```

Comment:
Would `set -e` do the same thing?

Reply:
yes, indeed. i had used the convention from https://bash3boilerplate.sh/ (they like the long form for readability), but i updated it just now to `set -e` (which is the form used in this repo).